scripts/error-codes/codes.json

```diff
@@ -104,11 +104,11 @@
   "102": "EventPluginRegistry: Cannot inject two different event plugins using the same name, `%s`.",
   "103": "executeDirectDispatch(...): Invalid `event`.",
   "104": "ReactCompositeComponent: injectEnvironment() can only be called once.",
-  "105": "%s(...): A valid React element (or null) must be returned. You may have returned undefined, an array or some other invalid object.",
+  "105": "%s(...): A valid React element (or null) must be returned, but you returned %s.",
```

Comment:
English nit: comma before `but` is not necessary? (I'm not actually a native speaker so correct me if I'm wrong.)